.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules/
 npm-debug.log
 build
+core
```

Comment:
It's the new lib. I am just sort of following material's standard. It does look nice instead of seeing lib.

e.g. 
`react-cm-ui/core/styles`
vs
`react-cm-ui/lib/styles`